examples/guestbook-go/v1beta3/guestbook-controller.json

```diff
@@ -0,0 +1,39 @@
+{
+    "kind": "ReplicationController",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "guestbook",
+        "labels": {
+            "name": "guestbook"
+        }
+    },
+    "spec": {
+        "replicas": 3,
+        "selector": {
+            "name": "guestbook"
+        },
+        "template": {
+            "metadata": {
+                "name": "guestbook",
```

Comment:
I think this name is ignored:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/controller/replication_controller.go#L70

I'm guessing we're not validating that name isn't specified due to legacy v1beta1/2 support.

Can you confirm, @smarterclayton?